ONSdigital/dp-csv-splitter
# feature/initial-build-pipeline

### What

Added `Dockerfile`, `Jenkinsfile` and deployment scripts. Also vendored external dependencies.

### How to review

* Check the build pipeline for this branch, it should be passing
* Check the application has been deployed to the correct environment with the correct configuration

### Who can review

Anyone but myself.
